### PR TITLE
fix(python): respect concurrency in worker

### PR DESCRIPTION
**Pull request by roggervalf**
fixes https://github.com/taskforcesh/bullmq/issues/2063

### CODE REVIEW

File: python/bullmq/worker.py [hunk 1]
```diff
@@ -59,21 +59,19 @@ async def run(self):
         token_postfix = 0
 
         while not self.closed:
-            if len(jobs) == 0 and len(self.processing) < self.opts.get("concurrency") and not self.closing:
+            while len(self.processing) < self.opts.get("concurrency") and not self.closing:
```
[the remark this answers is not in the document]
Comment (roggervalf) [reply python/bullmq/worker.py:62]: I can see it similar https://github.com/taskforcesh/bullmq/blob/master/src/classes/worker.ts#L395

[the remark this answers is not in the document]
Comment (manast) [reply python/bullmq/worker.py:62]: Yes, the difference is the asyncQueue that keeps the excessive number of calls. It waits until BRPOPLPUSH has timeout or returned a job id. It is a bit tricky to understand the code, but that's the idea.

[the remark this answers is not in the document]
Comment (roggervalf) [reply python/bullmq/worker.py]: at the end, the magic is handled by waiting attribute in worker class

**Review by manast — APPROVED** [review]
It looks correct to me.

**Pull request merged**